src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -612,19 +611,46 @@ static boolean isVariantInCodingRegion(final GencodeFuncotation.VariantClassific
                 outputFuncotations.add(gencodeFuncotation);
             }
             catch ( final FuncotatorUtils.TranscriptCodingSequenceException ex ) {
-                 logger.error("Unable to create a GencodeFuncotation on transcript " + transcript.getTranscriptId() + " for variant: " +
+                 logger.error("Problem creating a GencodeFuncotation on transcript " + transcript.getTranscriptId() + " for variant: " +
                         variant.getContig() + ":" + variant.getStart() + "-" + variant.getEnd() + "(" + variant.getReference() + " -> " + altAllele + "): " +
                          ex.getMessage()
                  );
-
-                 // TODO: Create a default annotation here:
-
+                logger.warn("Creating default GencodeFuncotation on transcript " + transcript.getTranscriptId() + " for problem variant: " +
+                                variant.getContig() + ":" + variant.getStart() + "-" + variant.getEnd() + "(" + variant.getReference() + " -> " + altAllele + ")");
+                outputFuncotations.add( createDefaultFuncotationsOnProblemVariant( variant, altAllele, gtfFeature, reference, transcript, version ) );
             }
         }
-
         return outputFuncotations;
     }
 
+    @VisibleForTesting
+    static final GencodeFuncotation createDefaultFuncotationsOnProblemVariant( final VariantContext variant,
```

Comment:
This method needs docs explaining what a "problem variant" is, and linking to the relevant github issues.

Reply:
Sounds good.  Fixed.